src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -23,30 +23,37 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
   <!-- Shared -->
   <ItemGroup>
-    <Compile Include="..\..\Common\src\System\Numerics\Hashing\HashHelpers.cs">
+    <Compile Include="$(CommonPath)\System\Numerics\Hashing\HashHelpers.cs">
       <Link>Common\System\Numerics\Hashing\HashHelpers.cs</Link>
     </Compile>
-    <Compile Include="System\Numerics\ConstantHelper.cs">
+    <Compile Include="$(CommonPath)\CoreLib\System\Numerics\ConstantHelper.cs">
       <AutoGen>True</AutoGen>
       <DesignTime>True</DesignTime>
       <DependentUpon>ConstantHelper.tt</DependentUpon>
+      <Link>System\Numerics\ConstantHelper.cs</Link>
     </Compile>
-    <Compile Include="System\Numerics\JitIntrinsicAttribute.cs" />
-    <Compile Include="System\Numerics\Register.cs">
+    <Compile Include="$(CommonPath)\CoreLib\System\Numerics\JitIntrinsicAttribute.cs">
+      <Link>System\Numerics\JitIntrinsicAttribute.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\CoreLib\System\Numerics\Register.cs">
       <AutoGen>True</AutoGen>
       <DesignTime>True</DesignTime>
       <DependentUpon>Register.tt</DependentUpon>
+      <Link>System\Numerics\Register.cs</Link>
     </Compile>
-    <Compile Include="System\Numerics\Vector.cs">
+    <Compile Include="$(CommonPath)\CoreLib\System\Numerics\Vector.cs">
       <AutoGen>True</AutoGen>
       <DesignTime>True</DesignTime>
       <DependentUpon>Vector.tt</DependentUpon>
+      <Link>System\Numerics\Vector.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\CoreLib\System\Numerics\Vector_Operations.cs">
+      <Link>System\Numerics\Vector_Operations.cs</Link>
     </Compile>
-    <Compile Include="System\Numerics\Vector_Operations.cs" />
   </ItemGroup>
   <!-- Carry a copy of MathF where not available -->
   <ItemGroup Condition="'$(IsPartialFacadeAssembly)' == 'true' OR $(TargetGroup.StartsWith('netstandard'))">
-    <Compile Include="..\..\Common\src\System\MathF.netstandard.cs">
+    <Compile Include="$(CommonPath)\System\MathF.netstandard.cs">
```

Comment:
We do need this file in CoreLib. This is just a glue for building portable version of the library.

Reply:
This wasn't an actual change to the MathF code.  I was just fixing up the path to use the common MSBuild property instead of relative paths.

Reply:
ok